salt/output/json_out.py

```diff
@@ -54,7 +54,7 @@ def output(data):
     '''
     try:
         if 'output_indent' not in __opts__:
-            return json.dumps(data, default=repr, indent=4)
+            return json.dumps(data, default=repr, indent=4, encoding='latin1')
```

Comment:
I am not sure if the result is valid (I had many problems getting an interative python shell on the windows box)
Result: "NameError: global name '__salt_system_encoding__' is not defined"

Just for information:
If i run "salt-call pkg.list_pkgs --out=json" on the minion itself I get the same error (utf8...)